dist/tools/doccheck/generic_exclude_patterns

```diff
@@ -1,10 +1,26 @@
+warning: Member ADC_DEV_NUMOF \(macro definition\) of
+warning: Member ADC_NUMOF \(macro definition\) of
 warning: Member BTN[0-9]_INT_FLANK \(macro definition\) of
 warning: Member BTN[0-9]_MASK \(macro definition\) of
 warning: Member BTN[0-9]_MODE \(macro definition\) of
 warning: Member BTN[0-9]_PIN \(macro definition\) of
 warning: Member BTN[0-9]_PORT \(macro definition\) of
 warning: Member BTN[0-9]_PRESSED \(macro definition\) of
 warning: Member BTN[0-9]_RELEASED \(macro definition\) of
+warning: Member CLOCK_CORECLOCK \(macro definition\) of
+warning: Member CONFIG_ZTIMER_[UM]*SEC_ADJUST_SET \(macro definition\) of
+warning: Member CONFIG_ZTIMER_[UM]*SEC_ADJUST_SLEEP \(macro definition\) of
+warning: Member CONFIG_ZTIMER_[UM]*SEC_DEV \(macro definition\) of
+warning: Member CONFIG_ZTIMER_[UM]*SEC_FREQ \(macro definition\) of
+warning: Member CONFIG_ZTIMER_[UM]*SEC_MIN \(macro definition\) of
+warning: Member CONFIG_ZTIMER_[UM]*SEC_TYPE \(macro definition\) of
+warning: Member CONFIG_ZTIMER_[UM]*SEC_WIDTH \(macro definition\) of
```

Comment:
there is only USEC

there shall never be any thing else because the USEC config of this is for backward compatibility

(these properties are timer properties)

Reply:
for `LPTIMER` the freq is `CONFIG_ZTIMER_LPTIMER_FREQ`
for RTT it is `RTT_FREQUENCY`
for RTC it is `FREQ_1HZ`

for the "basic"-timer it should be `CONFIG_ZTIMER_TIMER_FREQ` at it is called `CONFIG_ZTIMER_USEC_FREQ` for historic reasons